Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN mkdir -p /go/src/github.com/asecurityteam/asset-inventory-api
 WORKDIR $GOPATH/src/github.com/asecurityteam/asset-inventory-api
 COPY --chown=sdcli:sdcli . .
 RUN sdcli go dep
-RUN GO111MODULE=on GOFLAGS='-mod=vendor' go get github.com/gobuffalo/packr/v2/packr2
```

Comment:
Question: why was this breaking the build?

Reply:
the mod=vendor flag disables `go get` entirely. We need to go get this package so we can execute the packr2 command.

Reply:
TIL, thanks!